frontend/app/src/components/account-management/CreateAccount.vue

```diff
@@ -47,6 +47,11 @@
                 required
               >
               </v-text-field>
+              <v-checkbox
+                v-model="userPrompted"
+                class="create-account__boxes__user-prompted"
+                label="I understand that if I lose my password, I lose access to my data. I have created a backup of the password."
```

Comment:
How about adding a tiny bit more text:

"Rotki saves all user data locally. I understand that if I lose my password, I lose access to my data. I have created a backup of the password."